### ISSUE TITLE
Giving rewards to wrong person

### ISSUE THREAD

**Issue by BlueOrange775** [LinhyCZ/MOTDgd_Unturned#1]
The plugin is giving the rewards to the wrong person...

```
MOTDgd >> User Guest| cri98musati completed advertisement.
You have given 5000 levels of experience to Guest| SGFusion.

```

**Comment by LinhyCZ** [comment]
Fixed in newest version, get it form https://dev.rocketmod.net/plugins/motdgd-advertisement-for-your-unturned-server/

**Comment by BlueOrange775** [comment]
Issue is not fixed in build #9. Still persists

**Comment by LinhyCZ** [comment]
Unfortunately because of Unrturned behaviour (Player names in commands cannot contain space) there is no way around it. Use (steamid) instead of (player) for commands where you can use steamid, because that is unique identifier.
